tests/notifications_python_client/test_authentication.py

```diff
@@ -49,6 +56,26 @@ def test_token_should_fail_to_decode_if_wrong_key():
     assert str(err.value) == "Signature verification failed"
 
 
+@pytest.mark.parametrize('exc_class', [
```

Comment:
what's `exc`? Is there a more accessible word for it?

Reply:
Good point, will make this more readable